.ci/pipeline/build-and-test-win.yml

```diff
@@ -51,7 +51,7 @@ steps:
   - script: |
       call activate CB
       cd ..
-      call s\conda-recipe\run_test.bat s
+      call s\conda-recipe\run_test.bat s\

```

Comment:
Interesting. For my edification, could you explain why this change is necessary (may impact a PR of mine)

Reply:
It is to save path compatibility for custom path and non-set path. Our CI runs tests from `s` directory while conda recipe builder - from default path.